ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -56,11 +56,11 @@ class RNTrackPlayer: RCTEventEmitter, MediaWrapperDelegate {
             "STATE_STOPPED": AudioPlayerState.stopped,
             "STATE_BUFFERING": AudioPlayerState.buffering,
```

Comment:
Shouldn't the same be applied to the states? AFAIK, they are also not working properly